src/ec/curve25519/x25519.rs

```diff
@@ -109,7 +113,11 @@ fn x25519_ecdh(
         point: &ops::EncodedPoint,
         #[allow(unused_variables)] cpu_features: cpu::Features,
     ) {
-        #[cfg(all(not(target_os = "ios"), target_arch = "arm"))]
+        #[cfg(all(
+            target_arch = "arm",
+            not(target_os = "ios"),
+            not(target_os = "watchos")
```

Comment:
This probably needs a resolution in https://github.com/briansmith/ring/pull/1919 first but this isn't fully correct. The two 64-bit ARM Apple Watch targets have NEON.